### PR TITLE
Ended and detached stats for a track.

### PR DESCRIPTION
**Pull request by alvestrand**
These stats have been requested.

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -325,19 +325,32 @@
...
-          <dt>sequence<DOMString> trackIds</dt>
+          <dt>sequence&lt;DOMString&gt; trackIds</dt>
           <dd><p>This is the id of the stats object, not the
             <code>track.id</code>.</p></dd>
         </dl>
 
         <section id="mststats-dict*">
           <h3><dfn>RTCMediaStreamTrackStats</dfn> dictionary</h3>
+	  <p>An RTCMediaStreamTrackStats object represents the stats about a
+	  MediaStreamTrakc's connection to the PeerConnection object for which
```
Comment (vr000m) [inline webrtc-stats.html]: s/Trakc/Track

File: webrtc-stats.html [hunk 2]
```diff
@@ -325,19 +325,32 @@
...
+	  one calls getStats.</p>
+	  <p>It appears in the stats as soon as it is attached. If it is
+	    detached, it continues to appear, but with the "detached" member
+	    set to True.</p>
```
Comment (vr000m) [inline webrtc-stats.html]: We should be explicit what attached and detached mean. Is this tracked attached to/detached from an RtpSender/RtpReceiver or PeerConnection?

Comment (vr000m) [inline webrtc-stats.html]: oops missed the reference to PC before. However, should the property be set to True if it is detached from a RtpSender/RtpReceiver?

**Comment by alvestrand** [comment]
Fixes #7

File: webrtc-stats.html (hunk 1 above)
Comment (alvestrand) [inline webrtc-stats.html]: Spelling error: MediaStreamTrakc -> MediaStreamTrack.

**Comment by alvestrand** [comment]
@vr000m ready for an LGTM from you now?

**Comment by vr000m** [comment]
LGTM.

**Pull request merged**